plugins/woocommerce-admin/client/marketplace/components/my-subscriptions/table/table.tsx

```diff
@@ -71,6 +78,26 @@ export function InstalledSubscriptionsTable( props: {
 		},
 	];
 
+	if ( ! props.rows || props.rows.length === 0 ) {
+		const marketplaceBrowseURL = getNewPath( {}, MARKETPLACE_PATH, {} );
+		const noInstalledSubscriptionsHTML = createInterpolateElement(
+			__(
+				'No extensions or themes installed. <a>Browse the Marketplace</a>',
+				'woocommerce'
+			),
+			{
+				p: <p />,
+				a: <a href={ marketplaceBrowseURL }>your account</a>,
```

Comment:
```suggestion
				a: <a href={ marketplaceBrowseURL } />,
```

I think the link content is accidentally copied from another link – fortunately the interpolation doesn't replace our proper text, but we should change this.

Reply:
Nice spot! Resolved now. It annoys me that I need to add a linting ignore comment above links anytime we do this with `createInterpolateElement`. Either way, solved!